doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -1824,6 +1824,33 @@ below will fail immediately and won't print anything.
 In any case, the value returned by the fully applied quotation is an
 unspecified dummy Ltac1 closure and should not be further used.
 
+Returning values to Ltac1 from Ltac2 tactics is supported using the `ltac2val` quotation.
+
+.. prodn::
+   ltac_expr += ltac2val : ( @ltac2_expr )
```

Comment:
For productions that are defined through ml code rather than mlgs, we like to appear in `orderedGrammar` for completeness.  Would you add the production through `common.edit_mlg`, which will make it appear automatically in `ltac_expr`?  Then use merely ``` :n:`ltac2val : ( @ltac2_expr )` ``` here without the `.. prodn::`  (Admittedly, the presentation for `ltac2_expr` needs improvement.)

A short example would be nice.